internal/ffmpeg/convert.go

```diff
@@ -25,6 +25,7 @@ func AvcConvertCommand(fileName, avcName string, opt Options) (result *exec.Cmd,
 			"-i", fileName,
 			"-movflags", "faststart",
 			"-pix_fmt", "yuv420p",
+			// "-vf", "scale='-2:trunc("+opt.Resolution+"/2)*2'",
```

Comment:
Should I have this also apply to animated images or only to videos?

Reply:
That's a great question we should actually ask our users...